torchaudio/models/wavernn.py

```diff
@@ -324,3 +335,21 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
 
         # bring back channel dimension
         return x.unsqueeze(1)
+
+
+def wavernn(pretrained: bool = True, progress: bool = True, **kwargs: Any):
+    r"""WaveRNN model based on the implementation from
+    `fatchord <https://github.com/fatchord/WaveRNN>`_.
+    Args:
+        pretrained (bool): If True, returns a model pre-trained on LJSpeech
+        progress (bool): If True, displays a progress bar of the download to stderr
+    """
+    kwargs.setdefault("upsample_scales", [5, 5, 11])
+    kwargs.setdefault("n_classes", 30)
+    kwargs.setdefault("n_output", 128)
```

Comment:
Let's add those as [default](https://github.com/pytorch/audio/blob/master/torchaudio/models/wavernn.py#L230) to the main WaveRNN class instead, with the other defaults.

I'm assuming you took the defaults from [here](https://github.com/pytorch/audio/blob/master/examples/pipeline_wavernn/main.py#L83).